src/main/java/io/jsonwebtoken/JwtParser.java

```diff
@@ -27,6 +28,52 @@
     public static final char SEPARATOR_CHAR = '.';
 
     /**
+     * Sets the expected value for the iss registered claim. If the actual content of the JWT does not match
+     * (including if it's empty), then an exception will be thrown
+     *
+     * @param issuer the string representing the expected value for the iss (issuer) claim of the JWT
+     * @return the parser for method chaining.
+     */
+    JwtParser setIssuer(String issuer);
```

Comment:
When read the Javadoc was kind of confusing for me, shouldn't be something like:

Sets the expected value for the iss registered claim. If set to a non-null value, the ```iss``` claim in the JWT will be validated against it....